test/jdk/javax/swing/JButton/HtmlButtonImageTest/HtmlButtonImageTest.java

```diff
@@ -30,20 +30,16 @@
  * @run main HtmlButtonImageTest
  */
 
-import java.awt.Dimension;
-import java.awt.FlowLayout;
-import java.awt.Point;
-import java.awt.Robot;
+import java.awt.*;
```

Comment:
I prefer explicit imports rather than wildcard. In recent years, wildcard imports are usually replaced with explicit ones in the JDK code, including tests.